router.go

```diff
@@ -348,7 +348,10 @@ func (r *Router) setupDocs() {
 	if !r.mux.Match(chi.NewRouteContext(), http.MethodGet, r.OpenAPIPath()) {
 		r.mux.Get(r.OpenAPIPath(), func(w http.ResponseWriter, req *http.Request) {
 			w.Header().Set("Content-Type", "application/vnd.oai.openapi+json")
-			w.Write(spec.Bytes())
+
+			// Convert to JSON to take advantage of keys being sorted lexicographically
+			sBytes, _ := json.Marshal(spec)
```

Comment:
I think `spec.Bytes()` is [supposed to convert to JSON](https://pkg.go.dev/github.com/Jeffail/gabs/v2#Container.Bytes) - curious if you found some issue with that?

Reply:
Took another look and you're right in that it coverts to JSON before outputting the bytes. This was likely from me not understanding how the openapi generation worked initially. Reverted in [44bed11](https://github.com/danielgtaylor/huma/pull/71/commits/44bed11fb0b20f5d4dd281de41f3328aaa828db3).